src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
@gibson042 
> (we actually _still_ have a similar situation in the form of `div = document.createElement( "div" )`: [attributes/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/attributes/support.js), [css/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/css/support.js), [data/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/data/support.js), [event/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/event/support.js), [manipulation/support](https://github.com/jquery/jquery/blob/63f19a95b9bfffdc7b8800f89a9def768cc3aebd/src/manipulation/support.js))

These are all code pieces from `compat` where we assume the browser environment and don't support Node, browser extensions etc. All uses of `document` on `master` are via the variable or a property of `window`, this is enforced via JSHint.